tests/manual-test-cases/Group21-Registries/21-1-Whitelist.robot

```diff
@@ -93,3 +93,24 @@ Basic Whitelisting
     Should Contain  ${output}  Access denied to unauthorized registry
 
     Cleanup VIC Appliance On Test Server
+
+Configure Whitelist
```

Comment:
OK this test name is wrong and so are some of the comments in here because ... Idk I must've just not been thinking about it when I wrote this test with some of the commands from above. I'll fix the name of this test and the two comments below in the AM since they're just cosmetic / informational changes and don't affect the functionality of the test